src/config.py

```diff
@@ -0,0 +1,23 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+DEBUG = True
+TESTING = False
```

Comment:
I think that we need to add optional support for changing these variables in `.env`.